dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# A shell script to start the Node.js servers 
+# along with the mongod process, all of which
+# are necessary for RocketMeet-client to function.
+
+# Get the init system of the developer's platform.
+initSystem=`ps --no-headers -o comm 1`
+
+# Perform a check and start mongod.
+if [ "$initSystem" = "init" ]
+then
+    sudo service mongod start
+else 
+    sudo systemctl start mongod
+fi
+
+# Start RocketMeet-server
+npm run dev --prefix ../RocketMeet-server/ &
```

Comment:
Question: Won't the script pause here itself because of the socket created by nodejs and express?

Reply:
No.
The command in line 19 ends with a '&' which pushes the command into the background and allows the remainder of the script to continue.